### ISSUE TITLE
Generating lcov combined report throws NullPointerException

### ISSUE THREAD

**Issue by ddlees** [bazelbuild/bazel#10682]
> ATTENTION! Please read and follow:
> - if this is a _question_ about how to build / test / query / deploy using Bazel, or a _discussion starter_, send it to bazel-discuss@googlegroups.com
> - if this is a _bug_ or _feature request_, fill the form below as best as you can.

### Description of the problem / feature request:

When standing up a new Bazel project w/ a single (simple) Spring Boot "Hello World!" app w/ tests `bazel coverage --combined_report=lcov //...` fails with a `NullPointerException`. `bazel coverage //...` works as expected.

### Feature requests: what underlying problem are you trying to solve with this feature?

N/A

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

Using Bazel 2.0 or 1.2.1 on a new "Hello World!" java Spring Boot project (rules_jvm_external) w/ a minimal set of tests run `bazel coverage --combined_report=lcov //...`

### What operating system are you running Bazel on?

MacOS Mojave (darwin)

### What's the output of `bazel info release`?

- `release 2.0.0`
- `release 1.2.1`

### If `bazel info release` returns "development version" or "(@non-git)", tell us how you built Bazel.

N/A

### What's the output of `git remote get-url origin ; git rev-parse master ; git rev-parse HEAD` ?

N/A

###  Have you found anything relevant by searching the web?

Anh... sort of but it's all from 2018 and none of the suggestions worked anyway.

### Any other information, logs, or outputs that you want to share?

The problem can be found [here](https://github.com/bazelbuild/bazel/blob/2.0.0/src/main/java/com/google/devtools/build/lib/bazel/coverage/BazelCoverageReportModule.java#L115) caused by [this guy](https://github.com/bazelbuild/bazel/blob/2.0.0/src/main/java/com/google/devtools/build/lib/bazel/coverage/CoverageReportActionBuilder.java#L246) and possibly by [this guy](https://github.com/bazelbuild/bazel/blob/2.0.0/src/main/java/com/google/devtools/build/lib/bazel/coverage/CoverageArgs.java#L68) and this [doohickey](https://github.com/google/auto/tree/master/value).

In any case the `args` variable should probably be checked for a `null` value ;-)

```
INFO: Invocation ID: 1cd88f5e-41fc-4bbc-814f-c3cc017f5439
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/master.zip) = d4cf13b0e71fa2f271bcddec030e8c15e588bb8e49318d70c8eefbf959c76a5e
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "d4cf13b0e71fa2f271bcddec030e8c15e588bb8e49318d70c8eefbf959c76a5e"
DEBUG: Call stack for the definition of repository 'com_google_protobuf' which is a http_archive (rule definition at /private/var/tmp/_bazel_ddlees@us.ibm.com/72d0e8b638cf586b4ef328a2d3cc1a0b/external/bazel_tools/tools/build_defs/repo/http.bzl:292:16):
 - /redacted/WORKSPACE:206:1
INFO: Using default value for --instrumentation_filter: "//services/java/examples/springboot/src/main/java/hello[/:]".
INFO: Override the above default with --instrumentation_filter
DEBUG: Rule 'kafka_connect_ibmcos_sink' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1569581636 +0100"
DEBUG: Call stack for the definition of repository 'kafka_connect_ibmcos_sink' which is a new_git_repository (rule definition at /private/var/tmp/_bazel_ddleesus.ibm.com/72d0e8b638cf586b4ef328a2d3cc1a0b/external/bazel_tools/tools/build_defs/repo/git.bzl:182:22):
 - /redacted/WORKSPACE:319:1
DEBUG: Rule 'kafka_connect_couchdb' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1516629181 +0100"
DEBUG: Call stack for the definition of repository 'kafka_connect_couchdb' which is a new_git_repository (rule definition at /private/var/tmp/_bazel_ddleesus.ibm.com/72d0e8b638cf586b4ef328a2d3cc1a0b/external/bazel_tools/tools/build_defs/repo/git.bzl:182:22):
 - /redacted/WORKSPACE:281:1
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.getArgs(BazelCoverageReportModule.java:115)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.generateCoverageReportAction(CoverageReportActionBuilder.java:246)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.createCoverageActionsWrapper(CoverageReportActionBuilder.java:193)
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.createCoverageReportActionsWrapper(BazelCoverageReportModule.java:97)
	at com.google.devtools.build.lib.analysis.BuildView.createResult(BuildView.java:488)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:431)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:212)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:143)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:271)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:130)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:108)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:543)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:205)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:601)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:652)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

INFO: Elapsed time: 19.473s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (520 packages loaded, 13641 targets configured)
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.getArgs(BazelCoverageReportModule.java:115)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.generateCoverageReportAction(CoverageReportActionBuilder.java:246)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.createCoverageActionsWrapper(CoverageReportActionBuilder.java:193)
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.createCoverageReportActionsWrapper(BazelCoverageReportModule.java:97)
	at com.google.devtools.build.lib.analysis.BuildView.createResult(BuildView.java:488)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:431)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:212)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:143)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:271)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:130)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:108)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:543)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:205)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:601)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:652)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.getArgs(BazelCoverageReportModule.java:115)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.generateCoverageReportAction(CoverageReportActionBuilder.java:246)
	at com.google.devtools.build.lib.bazel.coverage.CoverageReportActionBuilder.createCoverageActionsWrapper(CoverageReportActionBuilder.java:193)
	at com.google.devtools.build.lib.bazel.coverage.BazelCoverageReportModule$1.createCoverageReportActionsWrapper(BazelCoverageReportModule.java:97)
	at com.google.devtools.build.lib.analysis.BuildView.createResult(BuildView.java:488)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:431)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:212)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:143)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:271)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:130)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:108)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:543)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:205)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:601)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:652)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
FAILED: Build did NOT complete successfully (520 packages loaded, 13641 targets configured)
```

**Comment by keith** [comment]
I think this is a dup of https://github.com/bazelbuild/bazel/issues/6450

Can you try the 2.1.0 release candidate and see if that works for you?

**Comment by ddlees** [comment]
TY! I can confirm this is fixed in `2.1.0`
